scality_manila_utils/utils.py

```diff
@@ -197,3 +197,24 @@ def nfs_mount(export_path):
             os.rmdir(mount_point)
         except OSError as e:
             log.warning("Unable to clean up temporary NFS root: %s", e)
+
+
+def is_stored_on_sofs(path):
+    """
+    Check if the given location is stored on a SOFS filesystem.
+
+    :param path: an absolute path, e.g `/ring/fs/samba_shares`
+    :type path: string
+    :rtype: boolean
+    """
+    with io.open('/proc/mounts') as mounts:
+        for mount in mounts.readlines():
```

Comment:
Yeah because mock.mock_open doesn"t work with `for mount in mounts`. I couldn't mock this properly so I had to do it this way.